rose/client/main.py

```diff
@@ -1,4 +1,6 @@
+import imp
 import logging
+import os.path
```

Comment:
We can replace this with "import os".

Reply:
os.path is the precise module. We are not using os